ui/app/AppLayouts/Chat/CommunityComponents/CreateChannelPopup.qml

```diff
@@ -263,7 +263,7 @@ StatusModal {
                                                                 channel.id,
                                                                 Utils.filterXSS(popup.contentComponent.channelName.text),
                                                                 Utils.filterXSS(popup.contentComponent.channelDescription.text),
-                      categoryId)
+                      channel.categoryId)
```

Comment:
The indentation seems messed up here on Github

Reply:
It is indeed weirdly indented in source.